br-webui/views/camera.liquid

```diff
@@ -482,6 +540,129 @@
 	
 	socket.emit('get v4l2 cameras'); // get v4l2 options
 	
+	//PATRICK IS WORKING
+	// Deal with stream types
+	/// Populate valid video stream types
+	function populateStreamTypeSelect() {
+		// Get combobox and add all options, the value of each option is the string pipeline
+		var streamTypeSelect = document.getElementById("streamTypeSelect")
+		for (const [streamType, content] of Object.entries(PIPELINES)) {
+			// Create option and add to combobox
+			var option = document.createElement("option");
+			option.innerHTML = streamType;
+			option.value = content.pipeline_elements.join(' ! ')
+			
+			streamTypeSelect.appendChild(option)
+		}
+		
+		// Menu under streamTypeSelect with all options for each stream type
+		var streamTypeMenu = document.getElementById('streamTypeMenu')
+		//UDP
+		{
+			// Create a parent item to enable visibity changes of the sub items
+			let udpMenu = document.createElement("item")
+			udpMenu.id = 'UDP-MENU'
+
+			let title = document.createElement("h4")
+			title.innerHTML = "Clients: "
+
+			let input = document.createElement("input")
+			input.id = 'UDP-CLIENTS'
```

Comment:
please make the id lowercase
https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html#:~:text=Always%20favor%20lowercase%2C%20for%20elements,%22)%20and%20maintain%20lowercasing%20throughout.